instrumentation/opentelemetry-instrumentation-openai/tests/cassettes/test_chat_completion.yaml

```diff
@@ -0,0 +1,97 @@
+interactions:
+- request:
+    body: '{"messages": [{"role": "user", "content": "Say this is a test three times"}],
+      "model": "gpt-4", "stream": false}'
+    headers:
+      accept:
+      - application/json
+      accept-encoding:
+      - gzip, deflate
+      connection:
+      - keep-alive
+      content-length:
+      - '112'
+      content-type:
+      - application/json
+      host:
+      - api.openai.com
+      user-agent:
+      - OpenAI/Python 1.47.0
+      x-stainless-arch:
+      - arm64
+      x-stainless-async:
+      - 'false'
+      x-stainless-lang:
+      - python
+      x-stainless-os:
+      - MacOS
+      x-stainless-package-version:
+      - 1.47.0
+      x-stainless-runtime:
+      - CPython
+      x-stainless-runtime-version:
+      - 3.11.5
+    method: POST
+    uri: https://api.openai.com/v1/chat/completions
+  response:
+    body:
+      string: !!binary |
+        H4sIAAAAAAAAAwAAAP//hJBBS8NAEIXv+RXDntuSpLXV3BQPIihSxItI2CbTZO1md81MoFr632W3
```

Comment:
I think we want ` "decode_compressed_response": True,` here because..
* it is easier to visually verify test assertions are coherent with the data
* it is much easier to verify the data doesn't accidentally have sensitive information in it

https://github.com/square/exchange/blob/d93f551cb40b336e362b5f7a263854d1f1cb5a03/tests/providers/conftest.py#L39

Reply:
totally agree, thanks for the example